src/components/pages/Triggers/TriggersList/AddTriggerModal/ModalSecondStep.tsx

```diff
@@ -0,0 +1,48 @@
+import {Form} from 'antd';
+
+import {Button, Text} from '@src/components/custom-antd';
+
+// import {ActionFormItems} from '@src/components/organisms';
+import Colors from '@styles/Colors';
+
+import {StepsEnum} from './AddTriggerModal';
+import {StyledButtonsContainer, StyledStepDescription} from './AddTriggerModal.styled';
+
+type ModalSecondStepProps = {
+  setCurrentStep: (step: StepsEnum) => void;
```

Comment:
Optionally: these steps could have `onBack`/`onFinish` callbacks instead of `setCurrentStep`. It will decouple the flow out of single steps.